terraform/validators.tf

```diff
@@ -228,7 +228,7 @@ data "template_file" "ecs_task_definition" {
     command                    = local.validator_command
     logstash                   = var.enable_logstash
     logstash_image             = var.logstash_image
-    logstash_version           = ":${var.logstash_version}"
+    logstash_version           = substr(var.logstash_version, 0, 6) == "sha256" ? "@${var.logstash_version}" : ":${var.logstash_version}"
```

Comment:
move this into the `locals` block like `safety_rules` does

Reply:
done.